src/Stratis.Bitcoin.Features.PoA/FederationManager.cs

```diff
@@ -58,7 +58,7 @@ public abstract class FederationManagerBase : IFederationManager
         protected List<IFederationMember> federationMembers;
 
         /// <summary>Protects access to <see cref="federationMembers"/>.</summary>
-        private readonly object locker;
+        protected readonly object locker;
```

Comment:
wooo very dangerous

Reply:
Why? As long as it's used for locking `federationMembers` we are fine. So hard to missuse it.